streamingpro-it/src/test/scala/tech/mlsql/it/LocalBaseTestSuite.scala

```diff
@@ -54,7 +54,8 @@ trait LocalBaseTestSuite extends FunSuite with SparkOperationUtil with BeforeAnd
     setupRunParams()
     copyDataToUserHome(user)
     TestManager.loadTestCase(new File(testCaseDirPath))
-    PlatformManager.getOrCreate.run(new ParamsUtil(runParams))
+    // Load built-in and external plug-ins and start PlatformManager
```

Comment:
Remove this since PR https://github.com/allwefantasy/mlsql/pull/1511  have resolved this issue already.